frontend-react/package.json

```diff
@@ -65,15 +65,21 @@
         "build-base-dev": "yarn compile-scss-dev && node scripts/build.js && yarn copy-404page",
         "start-js": "node scripts/start.js",
         "copy-404page": "cp build/index.html build/404.html",
-        "compile-scss-prod": "sass --load-path=./node_modules/@uswds/uswds/packages --no-source-map --style=compressed --quiet src/global.scss:src/content/generated/global.out.css",
-        "compile-scss-dev": "sass --load-path=./node_modules/@uswds/uswds/packages --embed-source-map --quiet-deps src/global.scss:src/content/generated/global.out.css",
+        "compile-scss-prod": "npm-run-all compile-global-scss-prod compile-content-scss-dev",
```

Comment:
should this be `compile-content-scss-prod` instead of `compile-content-scss-dev`?

Reply:
thanks for the callout, fixed